about/shared-responsibility.md

```diff
@@ -0,0 +1,67 @@
+# Shared Responsibility Model
+
+Many things must be done to successfully run a hub for a community.
+Some of them are content-focused, some are community-focused, others are infrastructure-focused.
+
+2i2c's aim is to **share responsibility for each hub** with the communities we serve. We do this by defining the responsibilities that are a good fit for the skills and goals of each organization.
+This "Shared Responsibility Model" is a useful way to understand what actions communities are still expected to perform under a service agreement with 2i2c.[^1]
+
+[^1]: This is inspired by the [Shared Responsibility Model](https://aws.amazon.com/compliance/shared-responsibility-model/) from Amazon Web Services.
+
+## Why follow this model
+
+We use a Shared Responsibility model because we believe that it leads to the best service for the communities that we serve.
+Our main goal is to ensure that each hub service is maximally impactful for the community it serves, and that it achieves this impact in the most efficient way possible.[^ironies-automation]
+
+[^ironies-automation]: Even when collaborating with engineering expertise in other organizations, we describe our service model in terms of areas of responsibility, rather than "tiers" of service that provide "burst capacity" or support only on an as-needed basis. This is because service "tiers" often leads to anti-patterns where support is needed from a person that is not empowered to be efforted in their duties (e.g., if they have been away from infrastructure for many months, and only after a series of escalations are needed to debug something). For more information on this, see [the Ironies of Automation](https://ckrybus.com/static/papers/Bainbridge_1983_Automatica.pdf) as well as [this post](https://blog.acolyer.org/2020/01/08/ironies-of-automation/) and [this post](https://www.thinkautomation.com/automation-advice/the-ironies-of-automation-explored/) explaining its relevance to technology and service delivery.
+
+To do that, we want to do two things:
+
+- Assign responsibilities that are well-suited to the skills and the interests of each group. This will boost the impact of each party.
+- Choose the operational model that is most well-matched to whomever is carrying out a responsibility. This will boost the efficiency of each party.
+
+Our challenge is to figure out which responsibiltiies should lie with the community, and which should lie with the 2i2c team, to strike this balance of impact and efficiency.
+
+We also follow this model because it helps us ensure the [Community Right to Replicate](https://2i2c.org/right-to-replicate) their infrastructure.
```

Comment:
```suggestion
We also follow this model because it helps us ensure that the community has the [Right to Replicate](https://2i2c.org/right-to-replicate) their infrastructure.
```